app/schedules.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  QUARTERLY: 'Q4',
+  MONTHLY: 'M12',
+  THREE_DAY_QUARTERLY: 'T4'
+}
```

Comment:
BIG fan of this.

Only issue I have is we have in our other services constant schedules but their names are simply `Q4`, `M12` and `T4`.

For consistency sake I would suggest changing these to those already defined.

HOWEVER, I propose that these constant names are ssooooo much better than merely the DAX schedule name and so I would suggest adding a tech debt ticket to update the constant schedules to these across all of our services.

Reply:
I did originally add them as `Q4`, `M12` and `T4` but felt like you they aren't that clear what they mean.  `Q4` and `M12` have some logic to them, but `T4` and perhaps others that we add don't.

But agree consistency is important, so I've changed them back.

Tech debt ticket created to update all of them to more meaningful names -> https://eaflood.atlassian.net/browse/SFI-2153